launch/ununifi-test-v1/memo_ununifi-test-v1.md

```diff
@@ -0,0 +1,299 @@
+```bash
+# validator-a
+> ~/go/bin/ununifid gentx validator-a 125000000000000uguu --chain-id="ununifi-test-v1" --from="validator-a" --ip="2001:19f0:7002:4d3:5400:04ff:fe07:ef3a" --moniker="ununifi-test-a" --identity="ununifi-test-a" --node-id="65710949120e28f8af12f81b75efd2a509280f70" --commission-rate=0.1 --commission-max-rate=1.0 --commission-max-change-rate=1.0 --min-self-delegation=1 --website="https://cauchye.com" --security-contact="info@cauchye.com" --details="CauchyE validator node. Delegate your tokens and Start Earning Staking Rewards."
+# validator-b
+> ~/go/bin/ununifid gentx validator-b 125000000000000uguu --chain-id="ununifi-test-v1" --from="validator-b" --ip="2001:19f0:7002:4d3:5400:04ff:fe07:ef3a" --moniker="ununifi-test-b" --identity="validator-b" --node-id="65710949120e28f8af12f81b75efd2a509280f70" --commission-rate=0.1 --commission-max-rate=1.0 --commission-max-change-rate=1.0 --min-self-delegation=1 --website="https://cauchye.com" --security-contact="info@cauchye.com" --details="CauchyE validator node. Delegate your tokens and Start Earning Staking Rewards."
+# validator-c
+> ~/go/bin/ununifid gentx validator-c 125000000000000uguu --chain-id="ununifi-test-v1" --from="validator-c" --ip="2001:19f0:7002:4d3:5400:04ff:fe07:ef3a" --moniker="ununifi-test-c" --identity="validator-c" --node-id="65710949120e28f8af12f81b75efd2a509280f70" --commission-rate=0.1 --commission-max-rate=1.0 --commission-max-change-rate=1.0 --min-self-delegation=1 --website="https://cauchye.com" --security-contact="info@cauchye.com" --details="CauchyE validator node. Delegate your tokens and Start Earning Staking Rewards."
+# validator-d
+> ~/go/bin/ununifid gentx validator-d 125000000000000uguu --chain-id="ununifi-test-v1" --from="validator-d" --ip="2001:19f0:7002:4d3:5400:04ff:fe07:ef3a" --moniker="ununifi-test-d" --identity="validator-d" --node-id="65710949120e28f8af12f81b75efd2a509280f70" --commission-rate=0.1 --commission-max-rate=1.0 --commission-max-change-rate=1.0 --min-self-delegation=1 --website="https://cauchye.com" --security-contact="info@cauchye.com" --details="CauchyE validator node. Delegate your tokens and Start Earning Staking Rewards."
+```
+
+# ununifid collect-gentxs
+```bash
+> ~/go/bin/ununifid collect-gentxs;
+```
+
+# add faucet account
+```bash
+> ~/go/bin/ununifid keys add faucet-a >& backup-faucet-a.txt
+> ~/go/bin/ununifid keys add faucet-b >& backup-faucet-b.txt
+> ~/go/bin/ununifid keys add faucet-c >& backup-faucet-c.txt
+> ~/go/bin/ununifid keys add faucet-d >& backup-faucet-d.txt
+```
+# Set balance for Faucet account
+```bash
+> ~/go/bin/ununifid add-genesis-account faucet-a 125000000000000uguu;
+> ~/go/bin/ununifid add-genesis-account faucet-b 125000000000000uguu;
+> ~/go/bin/ununifid add-genesis-account faucet-c 125000000000000uguu;
+> ~/go/bin/ununifid add-genesis-account faucet-d 125000000000000uguu;
+```
+
+# Start node.
+```bash
+> ~/go/bin/ununifid start
+# stop "Ctrl + c"
+> ~/go/bin/ununifid unsafe-reset-all
```

Comment:
I guess you know, unsafe-reset-all cmd will be remove all history data. Be careful to use it.